src/docs/development/ui/navigation/deep-linking.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deep linking
+description: Navigate to routes when the app receives a new URL
+---
+
+{{site.alert.note}}
+  This feature is only available on the dev or master channel. See [Switching
+  Flutter channels][switching-channels] for more information.
+{{site.alert.end}}
+
+Flutter supports deep linking on iOS, Android and web browsers in the dev
+channel. Users opening a URL will display a specific screen in your app. By
+following these steps, your can launch and display routes via named routes
+(either with the [`routes`][routes] parameter or
+[`onGenerateRoute`][onGenerateRoute]), or by using the [Router][] widget.
```

Comment:
I didn't understand "Users opening a URL will display a specific screen in your app." I tried to clarify, but I might have missed your intent.

Also, please stop using VIA. Engs just LOVE via, but it's against our developer writing guidelines.

```suggestion
Flutter supports deep linking on iOS, Android, and web browsers in the dev
channel. Opening a URL displays that screen in your app. With the
following steps, you can launch and display routes by using named routes
(either with the [`routes`][routes] parameter or
[`onGenerateRoute`][onGenerateRoute]), or by using the [`Router`][Router] widget.
```